content/content.go

```diff
@@ -8,20 +8,25 @@ import (
 	"github.com/opencontainers/go-digest"
 )
 
+// ReaderAt is a reader at a specifc position
```

Comment:
`// ReaderAt extends the standard io.ReaderAt interface with reporting of Size and io.Closer`.